FormidableLabs/victory-docs
# colors updated for a11y and to match code sample styles to zeplin

# Overview 
This PR is to update the colors used for code samples and also to clean up theme colors and make sure contrasts are OK for a11y. 

## `<code>` styles 
- The prism styles that control the `<code>` blocks and markdown (see docs/anything) hadn't been updated yet. Here we have updated the colors to be inline with zeplin. I had to darken the foreground colors and lighten the background pink on the large code blocks to comply with a11y, they look like this: 

This branch: 
<img width="1363" alt="Screen Shot 2020-02-21 at 11 04 02 AM" src="https://user-images.githubusercontent.com/10720454/75083077-03192180-54d4-11ea-8307-b5e6b815285c.png">

vs in Zeplin: 
<img width="316" alt="Screen Shot 2020-02-21 at 6 01 24 PM" src="https://user-images.githubusercontent.com/10720454/75083104-3196fc80-54d4-11ea-870d-d18d63c8e37d.png">

(mostly it's that teal that's noticeably different). 

## Theme colors 
- While updating the prism styles I got kinda annoyed with how many different browns there were and that it was hard to tell them apart, combine that with running an axe check on color contrast and getting 900 errors from the various links that weren't contrasted enough anddd here we are! 
- Updated theme color ordering to go from lightest to darkest shade of that color
- updated the generic color descriptions to be their true (or as close as possible) color name
- removed colors that were redundant or just sligghttlly different (looking at you `darkGray`)
- adjusted darkness to comply with a11y contrast standards

There's only really one noticeable color change here, and it's on the home page.

This branch: 
<img width="1413" alt="Screen Shot 2020-02-21 at 11 04 16 AM" src="https://user-images.githubusercontent.com/10720454/75083221-d6193e80-54d4-11ea-8f18-355b62937297.png">

Zeplin: 
<img width="330" alt="Screen Shot 2020-02-21 at 6 06 47 PM" src="https://user-images.githubusercontent.com/10720454/75083243-f21ce000-54d4-11ea-9251-cffcaa477242.png">

### Palettes for reference
<img width="905" alt="Screen Shot 2020-02-21 at 2 47 14 PM" src="https://user-images.githubusercontent.com/10720454/75083304-4e7fff80-54d5-11ea-8281-d9e5804b4ec3.png">
<img width="915" alt="Screen Shot 2020-02-21 at 2 48 22 PM" src="https://user-images.githubusercontent.com/10720454/75083305-4fb12c80-54d5-11ea-8260-7eed9ded6ded.png">
<img width="908" alt="Screen Shot 2020-02-21 at 2 58 16 PM" src="https://user-images.githubusercontent.com/10720454/75083307-5049c300-54d5-11ea-8c01-b8bc1efa2d39.png">

## To Test 
- pull and run code locally, see that markdown now follows the styles set in zeplin for single lines and blocks
